.github/workflows/helm-chart-tests.yaml

```
@@ -0,0 +1,823 @@
+          sed -i 's|$ChartRepoURL|https://falkordb.github.io/helm-charts/|g' ${{ matrix.plans.file }}
+          sed -i 's|$FalkorDBServiceVersion|${{ env.FALKORDB_SERVICE_VERSION }}|g' ${{ matrix.plans.file }}
+          sed -i 's/validPaymentMethodRequired: true/validPaymentMethodRequired: false/g' ${{ matrix.plans.file }}
+
+      - name: Upload yaml as artifact
+        uses: actions/upload-artifact@v4
+        with:
+          name: ${{ matrix.plans.file }}
+          path: compose/${{ matrix.plans.file }}
+
+      - name: Update Omnistrate plan
+        uses: ./.github/actions/update-omnistrate-plan
```

actionlint (1.7.9)

file "/bin/sh" does not exist in "/home/jailuser/git/.github/actions/update-omnistrate-plan". it is specified at "entrypoint" key in "runs" section in "Update Omnistrate Plan" action

(action)